src/playwright-runner.js

```diff
@@ -374,7 +374,10 @@ async function runPlaywright(nodeBin, runCfg) {
 
   // Copy our runner's playwright config to a custom location in order to
   // preserve the customer's config which we may want to load in the future
-  const configFile = path.join(runCfg.projectPath, 'sauce.config.js');
+  //
+  // Copy with .mjs extension in case customer package.json does not have
+  // ESM modules enabled.
```

Comment:
Should we rename the actual source file to `sauce.config.mjs`? Rather than rename-while-copying? 🤔